ale/drivers/cassini_drivers.py

```diff
@@ -175,3 +232,47 @@ def sensor_model_version(self):
           ISIS sensor model version
         """
         return 1
+
+    @property
+    def frame_chain(self):
+        if not hasattr(self, '_frame_chain'):
+            if self.instrument_id == 'CASSINI_ISS_NAC':
+                self._frame_chain = FrameChain.from_spice(frame_changes = [(1, self._original_naif_sensor_frame_id), (1, self.target_frame_id)], ephemeris_time=self.ephemeris_time)
+                rot_180 = Rotation.from_euler('z', 180, degrees=True)
+                self._frame_chain.add_edge(self._original_naif_sensor_frame_id, self.sensor_frame_id, ConstantRotation(rot_180.as_quat(), self._original_naif_sensor_frame_id, self.sensor_frame_id))
```

Comment:
Is it worth making the Rotation classes constructable from a scipy rotation?

Reply:
we definitely need to make this cleaner, seems better to do it in a different PR. Adam makes it seem like it's not as simple as just constructing ALE Rotations from Scipy Rotations. Kinda want to get these rotations as close to scipy rotations as possible (i.e. not having our own Rotation at all).